src/Compilers/CSharp/Portable/xlf/CSharpResources.cs.xlf

```diff
@@ -387,6 +387,11 @@
         <target state="new">SyntaxTree is not part of the compilation, so it cannot be removed</target>
         <note />
       </trans-unit>
+      <trans-unit id="WRN_TypeParameterSameAsParentFunctionParameter">
+        <source>Type parameter '{0}' has the same name as the type parameter from parent function '{1}'</source>
+        <target state="new">Type parameter '{0}' has the same name as the type parameter from parent function '{1}'</target>
```

Comment:
The messages in the .xlf files are stale.

Reply:
Will fix all now -- getting an error due to old .xlf files but I'm not sure where to run `msbuild /t:UpdateXlf` (says to in error but I tried in PackageManagerConsole and CLI and nothing)